src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -0,0 +1,136 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import {hide as hideSidebar} from '../../../../libs/actions/Sidebar';
```

Comment:
@roryabraham yep, we got rid of the `Sidebar.hide()` since there is no need to toggle the sidebar this way. Since we are using a drawer navigator (for now) I don't think we should really need to explicitly close the drawer. Navigating to a new route _should_ do this automatically.

- This will tell us if the drawer is open or not for the cases where we maybe have `isSidebarShown` before, but I'm not sure that's relevant to this PR.
https://github.com/Expensify/Expensify.cash/blob/c816a13b55b2b8b76c865a181371b1b083f5f62e/src/libs/Navigation/Navigation.js#L60-L67

Reply:
Thanks for clarifying!

@barun1997 as Marc said, this shouldn't be necessary here. But please let us know if you face any issues!

Reply:
@Julesssss  @marcaaron Got it! Thank you.